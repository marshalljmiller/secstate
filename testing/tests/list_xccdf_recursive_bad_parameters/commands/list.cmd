/usr/bin/secstate list -q
