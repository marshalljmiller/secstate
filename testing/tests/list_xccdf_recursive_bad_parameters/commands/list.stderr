Usage: secstate list [options] [ContentID]

secstate: error: no such option: -q
